templates/artist/artist_detail.html

```diff
@@ -36,49 +36,49 @@
 
 {% block extrastyle %}
     <link rel="stylesheet" href="https://code.jquery.com/ui/1.11.4/themes/smoothness/jquery-ui.css">
-    <link rel="stylesheet" href="{% static "css/artist/countdown.css" %}" type="text/css" charset="utf-8" />
     <link rel="stylesheet" href="{% static "css/artist/artist_detail.css" %}" type="text/css" charset="utf-8" />
-    <style>
-        {% include "artist/includes/artist_detail_countdown_style.html" with campaign=campaign %}
-        {% for past_campaign in artist.past_campaigns %}
-            {% include "artist/includes/artist_detail_countdown_style.html" with campaign=past_campaign %}
-        {% endfor %}
-    </style>
 {% endblock %}
 
 {% block content %}
-    <h1 class="header-font">{{ artist.name }}</h1>
+    <h2>{{ artist.name }}</h2>
     {% if investors or artist.past_campaigns or updates or has_permission_to_submit_update %}
         <ul class="tabs" data-tabs id="artist-detail-tabs">
-            <li class="tabs-title {% if not form.errors %}is-active{% endif %}"><a href="#overview" aria-selected="true">Overview</a></li>
-            {% if investors %}
-                <li class="tabs-title"><a href="#investors">Investors</a></li>
-            {% endif %}
-            {% if artist.past_campaigns %}
-                <li class="tabs-title"><a href="#past">Past Campaigns</a></li>
-            {% endif %}
-            {% if updates or has_permission_to_submit_update %}
-                <li class="tabs-title {% if form.errors %}is-active{% endif %}"><a href="#updates">Updates</a></li>
-            {% endif %}
+            <li class="tabs-title {% if not form.errors %}is-active{% endif %}"><a href="#overview" aria-selected="true"><i class="fa fa-user fa-2x tab-icon" aria-hidden="true"></i><span class="artist-tab">Overview</span></a></li>
+            <li class="tabs-title"><a href="#investors"><i class="fa fa-money fa-2x tab-icon" aria-hidden="true"></i><span class="artist-tab">Invest</span></a></li>
```

Comment:
Cool. I agree.

My concern was that if I changed any of these names that it would break something in the backend. If not, I'll go ahead and clean things up a bit more